qds_sdk/commands.py

```diff
@@ -436,6 +436,8 @@ class SparkCommand(Command):
 
     optparser.add_option("--sql", dest="sql", help="sql for Spark")
 
+    optparser.add_option("--note-id", dest="note_id", help="Id of the Notebook to run.")
```

Comment:
The model name in DB for storing notebooks in `notes`, so in order to map association, we have to name this field  as `note_id`.